### ISSUE TITLE
Update 'globalize' gem for Rails 5

### ISSUE THREAD

**Issue by olliebennett** [helpyio/helpy#237]
The [globalize](https://github.com/globalize/globalize) needs a release to support Rails 5.

**Comment by scott** [comment]
It looks like if you use `master` it will work- see https://github.com/globalize/globalize/issues/527

**Comment by olliebennett** [comment]
Yup; just waiting on a release - I think it would be best to avoid pointing `Gemfile` to master, although I'll do that in the Rails 5 PR so we can see what else breaks while we wait for a release.

**Comment by scott** [comment]
yeah thats what I was thinking too.

**Comment by connorshea** [comment]
https://github.com/globalize/globalize/issues/543#issuecomment-247559430

Could someone open a PR for helpy to make sure master of globalize doesn't break things?
